understanding/target-size-minimum.html

```diff
@@ -385,159 +293,111 @@ <h3>Spacing</h3>
-                        low vision may prefer large controls.
+                     <p class="note-title marker">注記</p>
+                     <p>異なる障害のある利用者は、コントロールのサイズに対するニーズも異なる。可視のターゲットのサイズを増やすことなく、アクティブなターゲット領域を増やすオプションを提供すると有益な可能性がある。もう一つのオプションは、レイアウトの密度を制御し、それによってターゲットのサイズもしくは間隔、又はその両方を変更するメカニズムを提供することである。これは幅広い利用者にとって有益な可能性がある。例えば、視野損失のある利用者は、より小さなサイズのコントロールを備えた、より圧縮されたレイアウトを好む場合がある一方で、他の形態のロービジョンの利用者は、大きなコントロールを好む場合がある。
                      </p>
                   </div>
                   
                </section>
             </section>
             <section id="benefits">
-               <h2>Benefits</h2>
-               <p>Having targets with sufficient size - or at least sufficient target spacing - can
-                  help all users who may have difficulty in confidently targeting or operating small
-                  controls. Users who benefit include, but are not limited to:
+               <h2>利点</h2>
+               <p>十分なサイズのターゲット (又は少なくとも十分なターゲットの間隔) を持つことは、小さなコントロールを自信を持ってターゲットにする又は操作することが困難な全ての利用者を助けることができる。メリットを受ける利用者には次が含まれるが、これらに限定されない:
                </p>
                <ul>
                   
-                  <li>People who use a mobile device where the touch screen is the primary mode of interaction;</li>
+                  <li>タッチスクリーンが主な操作方法であるモバイルデバイスを使用する人々。</li>
                   
-                  <li>People using mouse, stylus or touch input who have mobility impairments such as hand
-                     tremors;
+                  <li>マウス、スタイラスペン、又はタッチ入力を使用する、手の震えなどの運動障害のある人。
                   </li>
                   
-                  <li>People using a device in environments where they are exposed to shaking such as public
-                     transportation;
+                  <li>公共交通機関など、揺れにさらされる環境でデバイスを使用する人々。
                   </li>
                   
-                  <li>Mouse users who have difficulty with fine motor movements;</li>
+                  <li>細かい運動が困難なマウスの利用者。</li>
                   
-                  <li>People who access a device using one hand;</li>
+                  <li>片手でデバイスにアクセスする人々。</li>
                   
-                  <li>People with large fingers, or who are operating the device with only a part of their
-                     finger or knuckle.
+                  <li>指が大きい人々、又は指もしくは関節の一部だけでデバイスを操作している人々。
                   </li>
                   
                </ul>
             </section>
             <section id="examples">
-               <h2>Examples</h2>
+               <h2>事例</h2>
                <ul>
                   
-                  <li>Three buttons are on-screen and the target size of each button is 24 by 24 CSS pixels.
-                     Since the target size itself is 24 by 24 CSS pixels, no additional spacing is required,
-                     the Success Criterion passes.
+                  <li>画面上に三つのボタンがあり、各ボタンのターゲットのサイズは 24 × 24 CSS ピクセルである。ターゲットのサイズ自体が 24 × 24 CSS ピクセルであるため、追加の間隔は必要なく、達成基準は合格する。
```

Comment:
the Success Criterion passes. の訳ですが、若干、意訳気味に手を加えてみましたがいかがでしょうか。

```suggestion
                  <li>画面上に三つのボタンがあり、各ボタンのターゲットのサイズは 24 × 24 CSS ピクセルである。ターゲットのサイズ自体が 24 × 24 CSS ピクセルであるため、追加の間隔は必要なく、達成基準に合格する。
```